profile/README.md

```diff
@@ -2,40 +2,30 @@
-| :---:        |              :---:             |            :---:             |            :---:             |            :---:             | :---:        |
+| Quick links  |  [``` OpenEBS Website ```](https://www.openebs.io/)  |  [``` OpenEBS Docs ```](https://www.openebs.io/docs)  | [``` Main Repo ```](https://github.com/openebs/openebs) |  [``` Community Repo ```](https://github.com/openebs/community) |  [``` Community Meeting ```](https://github.com/openebs#monthly-community-meetings) |
 
 ---
 
-## Project Purpose
+## Overview
 
-OpenEBS is an open-source storage service for Kubernetes applications. OpenEBS manages the block storage and file systems based on the block storage for containers running in Kubernetes. Use OpenEBS for creating fast and resilient storage; with options for single-node, and replicated multi-node storage.
+### What is OpenEBS?
 
-OpenEBS is a Stateful Persistent block-mode Data Storage platform with a native virtual SAN fabric; for Kubernetes. We are a CNCF member project. We are the largest global community of K8s storage users.<BR>
-<BR>
-Our project team was an early pioneering inventor of K8s **Container Native Storage** services (we invented the term). We conceived the vision of a Stateful Persistent data platform for K8s that is tightly integrated and natively embedded into the core of K8s.<BR>
-<BR>
-We built an innovative High-performance Enterprise grade Block-mode Hyper-converged Storage virtual SAN Fabric that augments the storage services of K8s with Stateful Persistence, Enterprise Data mgmt, SSD/NVMe optimized I/O services, Replicated Data volumes, Thin Provisioning, Snapshot, Clones; and many other Mission critical data storage services that ```don't come``` in K8s out-of-the-box.<BR>
-<BR>
-
-> **OpenEBS is very popular :** <BR>
-> Live OpenEBS systems actively report back product telemetry each day, to our Global Analytics system (unless disabled by the user).<BR>
->
-> :rocket: &nbsp; OpenEBS is a very popular Storage ```Platform``` for Kubernetes <BR>
-> :star: &nbsp; We have over [9k GitHub Stars](https://star-history.com/#openebs/openebs&Date) <BR>
-> :floppy_disk: &nbsp; We have been used to deploy millions of <kbd>Volumes</kbd> <BR>
+OpenEBS is an open-source Container Native Storage solution that provides persistent storage for Kubernetes workloads. It enables dynamic provisioning of storage resources using containerized storage controllers, making it highly flexible and cloud-native. OpenEBS supports various storage engines, including LocalPVs for direct node storage and Replicated PV advanced data replication and resilience. It is desiganed to integrate seamlessly with Kubernetes, offering benefits like storage policies, reize, thin-provisiong, snapshots, and restore capabilities, making it an ideal choice for stateful applications.
```

Comment:
typo - resize

Reply:
Missing 'for' in "and Replicated PV for advanced data replication and resilience."
typo - designed
typo - thin-provisioning